pontoon/contributors/templates/contributors/profile.html

```diff
@@ -264,6 +274,42 @@ <h3 class="controls">
                 </div>
             </section>
 
+            </div>
+
+            <section id="contributions">
+                <div class="header clearfix">
+                    <h2 class="title">{{ title }}</h2>
+                    <div class="type-selector controls select">
+                        <div class="button selector">
+                            <div class="value">
+                                <span data-type="all_user_contributions" title="User translations and reviews">All user contributions</span>
+                            </div>
+                            <span class="icon fa fa-caret-down"></span>
+                        </div>
+                        <div class="menu" style="display: none;">
+                            <ul>
+                                <li class="clearfix">
+                                    <span data-type="user_translations" title="Translations submitted by the user">User translations</span>
```

Comment:
I changed them, because the ones in the spec weren't clear to me. :-)

E.g. We don't use the term "Submissions", "performed" and "received" sound unusual when used together with Review and "Submissions and reviews" isn't clear about excluding reviews of user's translations.

I tried to make thing clearer in the tooltip, but I understand that's not very visible. Maybe we should show the tooltip text underneath the chart?

Reply:
* The concept of "peer" doesn't exist anywhere. Funny enough, it's almost never a peer who reviews your work.
* "User review" doesn't say much: are those done by the user, or received by the user.

Submissions was used to make it clear that's something entered by the user, and translation can't be used.

Since this went through review internally and externally, I'd stick to the specs. If those turn out to be unclear in everyday use, changing them should be very quick.

Reply:
> Since this went through review internally and externally, I'd stick to the specs. If those turn out to be unclear in everyday use, changing them should be very quick.

Fair point. Updated the wording.